core/file_monitor.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+from database.connector import insert_file_change_events
+from core.alert import info
+from core.compatible import byte_to_str
+from core.time_helper import now
+
+
+def is_excluded(path, dirs):
+    """
+    if path is excluded in list of dirs/files
+
+    :param path: path to check for exclude
+    :param dirs: list of excludes
+    :return: Boolean
+    """
+    for directory in dirs:
+        if path.startswith(directory):
+            return True
+    return False
+
+
+class containerFilesHandler(FileSystemEventHandler):
+    def __init__(self):
+        self.log_filename = None
+        self.EXCLUDES = None
```

Comment:
Why is this in caps? All caps is the standard notation for Constant values. These are not constant values if are they are being set later.